generated/ecs/ecs_nested.yml

```diff
@@ -1351,9 +1351,9 @@ destination:
   group: 2
   name: destination
   nestings:
-  - as
-  - geo
-  - user
+  - destination.as
+  - destination.geo
+  - destination.user
```

Comment:
We may not want to address this in this PR, as this is already a great improvement. But here we're also missing `destination.user.group`.

This PR improves the situation when a field set gets nested deeper directly, like `vlan` => `network.inner.vlan`. But it doesn't yet address a nesting within another nesting like `group` => `user.group`, then `user` => `destination.user`...

Not sure if we need to address that for now though. Concretely, this array drives things like the "field reuse" section at the bottom [here](http://ecs_803.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-destination.html#_field_reuse_4), and perhaps simply saying that "user" is nested here is enough.

WDYT?

Reply:
I like the idea of explicitly listing out all the reused fields so the links to all the reusables in a fieldset are in one place.  I'll add that change in.